test/DataMCSpectraComparison/histosSimplified.py

```diff
@@ -0,0 +1,533 @@
+#!/usr/bin/env python
+
+
+# Set to True in order to select e-mu and dielectron final states
+Electrons = False
```

Comment:
I guess the default is dimuon for Electrons=false, but that's missing in the comment above.
So if Electrons = True selects dimuon, mixed, and dielectrion, please also make this the default to avoid errors.